app/templates/suppliers/details.html

```diff
@@ -139,5 +137,12 @@
     {% endcall %}
   {% endif %}
 
+  {% if supplier.dunsNumber %}
```

Comment:
I think VAT number should be moved to just above DUNS as per the prototype

Reply:
that's where it is now, right?